src/main/java/com/backbase/ct/bbfuel/data/CommonConstants.java

```diff
@@ -76,6 +76,7 @@ private CommonConstants() {
 
     // Product summary
     public static final String PROPERTY_PRODUCTS_JSON_LOCATION = "products.json";
+    public static final String PROPERTY_ARRANGEMENT_CURRENTACCOUNT_EXTERNALIDS = "arrangement.currentaccount.externalIds";
```

Comment:
Current account is two words so needs an underscore.

Reply:
Done